packages/server/src/tests/utilities/structures.ts

```diff
@@ -160,7 +162,7 @@ export function automationTrigger(
   return {
     id: utils.newid(),
     ...triggerDefinition,
-  }
+  } as AutomationTrigger
```

Comment:
How come this is required?

Reply:
Because here we're having to create a Trigger from the definition which isn't actually a fully formed AutomationTrigger. Once we get away from having to define the definitions like this and use the builder it will be much better.

Reply:
`triggerDefinition` might be anything, and sadly typing go nuts... I tried my best, but not sure if there is a cleaner way to solve it